packages/blade/src/utils/metaAttribute/metaAttribute.web.ts

```diff
@@ -1,6 +1,16 @@
-const metaAttribute = (name: string, value: string | undefined): Record<string, string> => {
-  if (!value) return {};
-  return { [`data-${name}`]: value };
+import { MetaConstants } from './metaConstants';
+
+const metaAttribute = ({
+  name,
+  testID,
+}: {
+  name?: string;
+  testID?: string;
+}): Record<string, string> => {
+  return {
+    ...(name ? { [`data-${MetaConstants.Component}`]: name } : {}),
+    ...(testID ? { [`data-testid`]: testID } : {}),
```

Comment:
A test for this as well please, asserting test id is added if provided and otherwise not existing.